deploy/k8s-operator/kube-trailblazer/helm-charts/staging/rag-llm-pipeline/values.yaml

```diff
@@ -2,24 +2,31 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 triton:
-  modelDirectory: "/zvonkok/model/llama2_13b_chat_hf_v1/"
+  modelPVC: "llama2"
+  quantization: "None"
   modelArchitecture: "llama"
   modelMaxInputLength: "3000"
   modelMaxOutputLength: "512"
   image: localhost:5000/llm-inference-server
   gpu:
     # MIG slice
     #type: "nvidia.com/mig-3g.40gb"
-    # time-slice 
+    # time-slice
     type: "nvidia.com/gpu"
     count: 1
 
 milvus:
   gpu:
-    # MIG slice 
+    # MIG slice
     #type: "nvidia.com/mig-2g.20gb"
     type: "nvidia.com/gpu"
     count: 1
+  etcd:
```

Comment:
Can you add details to setup default `StorageClass` and a local provisioner for the PVC creation.